pages/models.py

```diff
@@ -464,40 +468,64 @@ def serve(self, request):
 # News / Statements / Press Releases
 
 class NewsIndex(Page):
-    social_image = models.ForeignKey('wagtailimages.Image', null=True, blank=True, on_delete=models.SET_NULL, related_name='+')
+    social_image = models.ForeignKey(
+        'wagtailimages.Image',
+        null=True,
+        blank=True,
+        on_delete=models.SET_NULL,
+        related_name='+'
+    )
     parent_page_types = ['pages.IndexPage']
     subpage_types = ['pages.NewsPost']
 
     promote_panels = Page.promote_panels + [
-            ImageChooserPanel('social_image')
-        ]
+        ImageChooserPanel('social_image')
+    ]
+
+    '''
+    Include all paged urls in cached paths
+    http://docs.wagtail.io/en/v1.10.1/reference/contrib/frontendcache.html
+    '''
+    def get_cached_paths(self):
```

Comment:
@cjmabry is it possible this method override is unnecessary, and/or caused a regression? did previously clearing cache on new index page by republishing also clear cache for all paginated pages? right now publishing news index page does not seem to clear news index page cache.

Reply:
fix: https://github.com/Our-Revolution/site/pull/66